### PR TITLE
feat: cssinjs for image

### PR DESCRIPTION
**Pull request by heiyu4585**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     image组件改为 cssinjs      |
| 🇨🇳 Chinese |    cssinjs for image       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .github/workflows/compressed-size.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
       CI_JOB_NUMBER: 1
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
```
Comment (zombieJ) [inline .github/workflows/compressed-size.yml:14]: 混入了其他的 PR

File: components/_util/theme/interface.ts [hunk 2]
```diff
@@ -14,6 +14,8 @@ export const PresetColors = [
   'geekblue',
   'lime',
   'gold',
+  'black',
```
Comment (zombieJ) [inline components/_util/theme/interface.ts:17]: black 和 white 直接 hardcode 到 image style 里去，Seed 和 Der 先不动

File: components/image/style/index.tsx [hunk 3]
```diff
@@ -1,2 +1,334 @@
...
+    insetBlockStart: 0,
+    insetInlineEnd: 0,
+    zIndex: 1,
+    display: 'flex',
+    flexDirection: 'row-reverse',
+    alignItems: 'center',
+    width: '100%',
+    color: black,
+    listStyle: 'none',
+    background: new TinyColor(modalMaskBg).setAlpha(0.1).toRgbString(),
```
Comment (zombieJ) [inline components/image/style/index.tsx:93]: 加一个 `FIXME`

File: components/_util/theme/interface.ts (hunk 2 above)
Comment (heiyu4585) [reply components/_util/theme/interface.ts]: done

**Comment by zombieJ** [comment]
CI failed. 看看是不是漏了 `// deps-lint-skip-all`

**Pull request merged**